RoddyCore/src/de/dkfz/roddy/config/ToolEntry.java

```diff
@@ -49,12 +49,20 @@ public ToolConstraint clone() {
         public boolean equals(Object o) {
             if (this == o) return true;
```

Comment:
Use explicit reference equality (also below; just in case we transform this later into Groovy, where == would result in infinite recursion.